doc/user/content/sql/create-sink/kafka.md

```diff
@@ -430,6 +431,41 @@ message delivery, you should ensure that:
 
 For more details, see [the Kafka documentation](https://kafka.apache.org/documentation/).
 
+### Partitioning
+
+{{< warn-if-unreleased "v0.113" >}}
+
+By default, Materialize assigns a partition to each message using the following
+scheme:
+
+  1. Encode the message's key in the specified format.
+  2. If the format uses a Confluent Schema Registry, strip out the
+     schema ID from the encoded bytes.
```

Comment:
```suggestion
  2. If the format uses a schema registry, strip out the schema ID from
    the encoded bytes.
```

Reply:
We consistently refer to the schema registry as a "Confluent Schema Registry" in the syntax and these docs—seems worth being consistent?

Reply:
Seems to be a mix right now. In the `CREATE SOURCE` page, we use "schema registry" (which I prefer, because it generalizes...even though the syntax doesn't). Cool to leave it as is here, but we should decide on one term and stick with it across the board.